content_scripts/mode_find.coffee

```diff
@@ -178,7 +178,10 @@ class FindMode extends Mode
       # ignore the selectionchange event generated by find()
       document.removeEventListener("selectionchange", @restoreDefaultSelectionHighlight, true)
 
-    result = window.find(query, options.caseSensitive, options.backwards, true, false, true, false)
+    try
+      result = window.find(query, options.caseSensitive, options.backwards, true, false, true, false)
+    catch # Failed searches throw on Firefox.
+    options.postFindFocus?.focus()
```

Comment:
Yes. `window.find` in Firefox focuses the window that we call it on, so this gives us an opportunity to refocus the HUD window.

Should I add a newline between the `try ... catch` and this line, to clearly separate them?